04-onion-routing.md

```diff
@@ -1407,6 +1408,19 @@ The _origin node_:
   - MAY use the data specified in the various failure types for debugging
   purposes.
 
+## `max_htlc_cltv` Selection
+
+If the HTLC expires too far in the future, a forwarding node can return an `expiry_too_far` error.
+
+This BOLT4 processing requirement comes with two downsides:
+- a payer is constrained on the maximum size of its payment path
+- a forwarding node is constrained on the `cltv_expiry_delta` for the safety of its own channels
+
+A common value between implementations allow forwarding nodes to increase their `cltv_expiry_delta`
+without reducing the payers ability to route along the network.
+
+This new common value `max_htlc_cltv` is suggested to be 4032 blocks.
```

Comment:
Looks like we all do 2016 today, so let's stick with that as the recommendation?

Reply:
Yep if we’re all doing 2016 today let’s stick to it. Good starting point then for a base value for routing hop gossips explicitly diverging from it.